datasource/src/DataSource.ts

```diff
@@ -107,8 +110,17 @@ export class DataSource extends DataSourceApi<SignalKQuery, SignalKDataSourceOpt
     const result = new Observable<DataQueryResponse>((subscriber) => {
       let lastStreamingValueTimestamp = 0;
 
-      const dataframe = new DualDataFrame(options.targets.map(
-        target => `${target.path}:${target.aggregate}`), 1000);
+      // const dataframe = new DualDataFrame(options.targets.map(
+      //   target => `${target.path}:${target.aggregate}`), 1000);
```

Comment:
I guess these commented rows aren't needed anymore

Reply:
Removed in 96f8ae97dcf5c0dcacf6d3df05ec3c9f9b804d69